tests/test_data_vis.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+from straPy.data_vis import histogram_ci_plot
+from pytest import raises
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def test_histogram_ci_plot():
+    """
+    Tests the histogram_ci_plot function to make sure the outputs are correct.
+        
+    Returns
+    --------
+    None
+        The test should pass and no asserts should be displayed.
+        
+    6 tests in total
+    """
+    
+    # tests with invalid input type of title
+    with raises(TypeError) as e:
+        histogram_ci_plot([1, 2, 3, 4, 5, 6, 7], 1000 ,n=100, ci_level=0.95, title = 123)
```

Comment:
```suggestion
        histogram_ci_plot([1, 2, 3, 4, 5, 6, 7], 1000, n=100, ci_level=0.95, title=123)
```

Reply:
@zackt113 can you add this small formatting change everywhere?

Reply:
Fixed it! Thanks @gauthampughazhendhi